### PR TITLE
Makefile: support build and download releated tools

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
1. fix the issue https://github.com/chaos-mesh/chaosd/issues/129, tell developers how to build && download chaosd related tools by using Makefile
2. update the Readme by the way. Delete the usage of chaosd, refer to the document.

### CODE REVIEW

File: .github/workflows/release_latest.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Prepare build environment
         run: |
           # actions/checkout require git v2.X
-          yum -y install https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm
```
Comment (WangXiangUSTC) [inline .github/workflows/release_latest.yml:23]: this URL is unavailable

File: README.md [hunk 2]
```diff
@@ -36,338 +40,46 @@ You can either build directly from the source or download the binary to finish t
...
-    Sample usage:
-
-    ```bash
-    curl -X POST "127.0.0.1:31767/api/attack/disk" -H "Content-Type: application/json" -d '{"action":"fill", "size":1024, "path":"temp", "fill-by-fallocate": true}' //filling using fallocate
-    ```
+    Then uncompress the archived file, and you can go into the folder and execute chaosd：
 
     ```bash
-    curl -X POST "127.0.0.1:31767/api/attack/disk" -H "Content-Type: application/json" -d '{"action":"fill", "size":1024, "path":"temp", "fill-by-fallocate": false}' //filling by writing data to files
+    tar zxvf chaosd-latest-linux-amd64.tar.gz && cd chaosd-latest-linux-amd64
```
Comment (cwen0) [inline README.md:80]: We also need to add a step to guide users to add `chaosd` to `PATH`.

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Comment by WangXiangUSTC** [comment]
@cwen0 PTAL again

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**